lib/go-atscfg/remapdotconfig_test.go

```diff
@@ -4663,6 +4768,110 @@ func TestMakeRemapDotConfigEdgeRangeRequestBGFetch(t *testing.T) {
+		49: map[string]string{
+			"cachekeykey": "cachekeyval",
+		},
+		44: map[string]string{
+			"shouldnotincludeotherprofile": "shouldnotincludeotherprofileval",
+		},
+	}
+
+	serverPackageParamData := map[string]string{
+		"dscp_remap_no": "notused",
+	}
+
+	serverInfo := &ServerInfo{
+		CacheGroupID:                  42,
+		CDN:                           "mycdn",
+		CDNID:                         43,
+		DomainName:                    "mydomain",
+		HostName:                      "myhost",
+		HTTPSPort:                     12443,
+		ID:                            44,
+		IP:                            "192.168.2.4",
+		ParentCacheGroupID:            45,
+		ParentCacheGroupType:          "CGType4",
+		ProfileID:                     46,
+		ProfileName:                   "MyProfile",
+		Port:                          12080,
+		SecondaryParentCacheGroupID:   47,
+		SecondaryParentCacheGroupType: "MySecondaryParentCG",
+		Type:                          "EDGE",
+	}
+
+	remapDSData := []RemapConfigDSData{
+		RemapConfigDSData{
+			ID:                       48,
+			Type:                     "HTTP_LIVE_NATNL",
+			OriginFQDN:               util.StrPtr("myorigin"),
+			MidHeaderRewrite:         util.StrPtr("mymidrewrite"),
+			CacheURL:                 util.StrPtr(""),
+			RangeRequestHandling:     util.IntPtr(tc.RangeRequestHandlingSlice),
+			CacheKeyConfigParams:     map[string]string{"cachekeyparamname": "cachekeyparamval"},
+			RemapText:                util.StrPtr("myremaptext"),
+			EdgeHeaderRewrite:        nil,
+			SigningAlgorithm:         util.StrPtr("foo"),
+			Name:                     "mydsname",
+			QStringIgnore:            util.IntPtr(int(tc.QueryStringIgnoreIgnoreInCacheKeyAndPassUp)),
+			RegexRemap:               util.StrPtr(""),
+			FQPacingRate:             util.IntPtr(0),
+			DSCP:                     0,
+			RoutingName:              util.StrPtr("myroutingname"),
+			MultiSiteOrigin:          util.StrPtr("mymso"),
+			Pattern:                  util.StrPtr(`mypattern`),
+			RegexType:                util.StrPtr(string(tc.DSMatchTypeHostRegex)),
+			Domain:                   util.StrPtr("mydomain"),
+			RegexSetNumber:           util.StrPtr("myregexsetnum"),
+			OriginShield:             util.StrPtr("myoriginshield"),
+			ProfileID:                util.IntPtr(49),
+			Protocol:                 util.IntPtr(int(tc.DSProtocolHTTPToHTTPS)),
+			AnonymousBlockingEnabled: util.BoolPtr(false),
+			Active:                   true,
+			SlicePluginBlockSize:     util.IntPtr(262144),
+		},
+	}
+
+	txt := MakeRemapDotConfig(serverName, toToolName, toURL, atsMajorVersion, cacheURLConfigParams, dsProfilesCacheKeyConfigParams, serverPackageParamData, serverInfo, remapDSData)
+
+	txt = strings.TrimSpace(txt)
+
+	testComment(t, txt, string(serverName), toToolName, toURL)
+
+	txtLines := strings.Split(txt, "\n")
+
+	if len(txtLines) != 2 {
+		t.Fatalf("expected 1 remaps from HTTP_TO_HTTPS DS, actual: '%v' count %v", txt, len(txtLines))
```

Comment:
same as above

Reply:
this is fatal?

Reply:
Sure looks that way. It's rare that the reviewer has copy/paste errors and the code they're reviewing doesn't.